utils.py

```diff
@@ -43,7 +47,74 @@ def server_response_decode(message):
             remainder = message_parts[1][b_string_len + 2 :]
 
     elif message[0] == ":":  # responses should start with '(integer)'
-        print("integer")
-        value = int(message_parts[0])
+        value = f"(integer) {message_parts[0]}"
         remainder = message_parts[1]
     return value, remainder
+
+
+def generate_redis_formatted_array(key_or_value, cli_message):
```

Comment:
it looks like this is only used in the client and should live there instead

Reply:
Function has packed its bags and moved.